Add pull_request_template.md (#1106)
rweekly/rweekly.org: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md

# Adding new content to R Weekly issue

Please include a one-sentence summary of the new content you would like to be added to our R Weekly issue.

## Type of Content

**Please select to which section(s) your post belongs!**

> <!--Please don't add anything to the Highlight section, R Weekly members vote for the content of this section weekly--> 

Reviewer: ```suggestion
<!--Please don't add anything to the Highlight section, R Weekly editors vote for the content of this section weekly--> 
```

- [ ] Tutorials - R tutorials for how to use certain packages and tools (usually code is embedded)
- [ ] Insights - Articles that talk about R and data science in general (usually no code embedded)
- [ ] R in Real World - Posts that discuss analyses that use R to analyze real-world data sets
- [ ] R in Organization - R use cases/events that showcase how organizations are utilizing or integrating R
- [ ] R in Academia - R use cases that showcase how Academia is utilizing R
- [ ] International - Non-English R related content
- [ ] Videos and Podcasts - Videos and Podcasts about R
- [ ] Resources - long posts, websites, books, slides, list, cheat sheets, or other learning resources in general that are more officially aggregated as a guide material
- [ ] Jobs - R Jobs
- [ ] New Packages and Tools - New packages and tools that have been created or published in the past two weeks.
- [ ] Updated Packages - New releases of tools and packages for R
- [ ] Call for Participation - New R groups, communities or competitions here. 
- [ ] Upcoming Events - Interesting R-related events or call for Participation section.

### I'd like to propose an Image from my new content!

> <!--Although you are very welcome to suggest an image, please bear in mind that images are added and changed by the editor in charge of the issue so we can guarantee that the image will be added--> 

Reviewer: ```suggestion
<!--Although you are very welcome to suggest an image, please bear in mind that images are added and changed by the editor in charge of the issue so we can't guarantee that your image will be added--> 
```

> Please feel free to suggest an image from your content to be added in the next issue

Reviewer: We could maybe add the suggestion of max width 600px

Author: Thank you, Jonathan, I changed the wording. Is this alright?

- [ ] Yes, I proposed an image and resized it
- [ ] No

# Checklist:

- [ ] My content is R-related 
- [ ] All images suggested are re-sized before has been added to the PR

> - [ ] I've submitted my content between Monday-Friday to allow for it to be voted by R Weekly members for highlights

Reviewer: ```suggestion
- [ ] I've submitted my content between Monday-Friday to allow for it to be voted by R Weekly editors for highlights
```
